src/goose/toolkit/utils.py

```diff
@@ -19,3 +22,23 @@ def get_language(filename: Path) -> str:
         return lexer.name
     except ClassNotFound:
         return ""
+
+
+def render_template(template_path: Path, context: Optional[dict] = None) -> str:
+    """
+    Renders a Jinja2 template given a Pathlib path, with no context needed.
+
+    :param template_path: Path to the Jinja2 template file.
+    :param context: Optional dictionary containing the context for rendering the template.
+    :return: Rendered template as a string.
+    """
+    # Ensure the path is absolute and exists
+    if not template_path.is_absolute():
+        template_path = template_path.resolve()
+
+    if not template_path.exists():
+        raise FileNotFoundError(f"Template file {template_path} does not exist.")
+
+    env = Environment(loader=FileSystemLoader(template_path.parent))
```

Comment:
[nit] could we look up parent paths recursively up until we reach the repo root? this is because in a monorepo setting it might be included at a subproject level, or at the top level of the repo - and goose could be ran at either level.

Reply:
i think this should be okay cause it just uses the working directory hints. i am gonna hold off on this till for now